client/modules/i18n/currency.js

```diff
@@ -34,20 +63,19 @@ export function formatPriceString(formatPrice) {
     try {
       // we know the locale, but we don"t know exchange rate. In that case we
       // should return to default shop currency
-      if (typeof locale.currency.rate !== "number") {
+      if (!Match.test(userCurrency.rate, Number)) {
```

Comment:
We thought consistency, but in this case it won't matter much. We'll put typeof back ;)